test/dlc_tests/eks/eks_manifest_templates/tensorflow/inference/single_node_neuron_inference.yaml

```diff
@@ -57,13 +57,15 @@ spec:
         - name: MODEL_BASE_PATH
           value: <MODEL_BASE_PATH>
         - name: AWS_REGION
-          value: us-west-2
+          value: us-east-1
```

Comment:
why is this region changed?

Reply:
Has the region been changed because there is a larger capacity for p3.16 instances in us-east-1?

Reply:
+1

Reply:
This is to fix single node neuron test. The bucket exist in `us-east-1` region. Reverting the change as it will be covered in #1376 PR.